internal/handler/msg_createuser.go

```diff
@@ -29,6 +37,116 @@ func (h *Handler) MsgCreateUser(ctx context.Context, msg *wire.OpMsg) (*wire.OpM
 		return nil, lazyerrors.Error(err)
 	}
 
-	// TODO https://github.com/FerretDB/FerretDB/issues/1491
-	return nil, common.Unimplemented(document, document.Command())
+	dbName, err := common.GetRequiredParam[string](document, "$db")
+	if err != nil {
+		return nil, err
+	}
+
+	if dbName != "$external" && !document.Has("pwd") {
+		return nil, handlererrors.NewCommandErrorMsg(
+			handlererrors.ErrBadValue,
+			"Must provide a 'pwd' field for all user documents, except those with '$external' as the user's source db",
+		)
+	}
+
+	// https://www.mongodb.com/docs/manual/reference/command/createUser/
```

Comment:
Hm, do we need this link here? We don't have links in the implementation of other commands.